pkg/utils/pod.go

```diff
@@ -174,6 +199,18 @@ func GetContainerNameFromPod(pod *v1.Pod, containerId string) string {
 		}
 	}
 
+	// for containerd
+	for _, v := range pod.Status.ContainerStatuses {
+		strList := strings.Split(v.ContainerID, "//")
+		if len(strList) > 0 {
+			klog.V(6).Infof("cri-containerd is %s ", "cri-containerd-"+strList[len(strList)-1]+".scope")
+			klog.V(6).Infof("containerid is %s", containerId)
+			if "cri-containerd-"+strList[len(strList)-1]+".scope" == containerId {
```

Comment:
"cri-containerd-"+strList[len(strList)-1]+".scope"
change to fmt.Sprintf?

Reply:
and did you use the same method from kubelet to find the containerId?

Reply:
OK，I didn't found codes related in kubelet, there is only some code like findContainerIDByName. And i search in github, https://github.com/search?p=1&q=cri-containerd-+.scope&type=Code , many projects do like us.